### DISCUSSION TITLE
Understanding workflow for station locations and LOTW/eQSL

### DISCUSSION THREAD

**Discussion by devinberg** [magicbug/Cloudlog#1531] [Q&A] [answered]
I'm trying to understand the proper workflow for dealing with multiple station locations (for example, when doing POTA activations). [From the wiki](https://github.com/magicbug/Cloudlog/wiki/LoTW-Import-&-Export-Documentation#qso-uploading) it says:

> Cloudlog builds the TQ8 file based on data you provide with the Station Location area, so if you are operating from a new area, just create the profile and when the script is run to upload to LoTW it will check that the Station Location has a valid certificate and upload, all on the fly no messing around.

Does this mean that a new station location needs to first be created in TQSL and then a new `.p12` file is created and uploaded? Or does the original `.p12` file cover a new location? Similarly, I assume that the new location needs to be created at eQSL.cc before syncing the log.

I guess what I am unclear on is what steps need to occur to upload logs correctly to LOTW and eQSL when I log from a new location.

**Comment by magicbug** [comment, marked as answer]
LoTW

- Upload p12 file from TQSL
- Create Station Location in Cloudlog
- Setup Cronjob to run the PHP script per install info
- If the Station Location callsign matches the p12 callsign stored in Cloudlog it will automatically upload to LoTW.

Cloudlog doesn't use TQSL at all for uploading QSOs its all handled by code I wrote myself to replace it.

eQSL

It doesn't handle multiple locations you have to have separate accounts as each nickname you create has to have a start and end date/time.. features are coming to support more than one account soon for version 2.

**Reply by devinberg** [reply]
Ok, so just to be clear, only one p12 file is needed regardless of how many station locations I add in CloudLog? I'm guessing then that the "Create Station Location" functionality in TQSL creates the tq8 files and this process has been replaced by CloudLog.

**Reply by magicbug** [reply]
Correct

**Reply by Ciemon** [reply]
Does that mean that for station profiles that use $callsign/P that we need to do something else if the QSOs under that location are to sync?
